tests/test_app.py

```diff
@@ -8,6 +9,20 @@ def test_init():
     assert app
 
 
+def test_run():
```

Comment:
Test names have to be self-explanatory. What about `test_run_calls_run_internally`?